apps/worker/src/app/workflow/usecases/send-message/send-message-in-app.usecase.ts

```diff
@@ -180,7 +182,35 @@ export class SendMessageInApp extends SendMessageBase {
       }),
     });
 
-    const bridgeBody = command.bridgeData?.outputs.body;
+    // V2 data
+    const bridgeOutputs = command.bridgeData?.outputs as InAppOutput;
+    const bridgeSubject = bridgeOutputs.subject;
+    const bridgeBody = bridgeOutputs.body;
+    const bridgeCta = {
+      action: {
+        buttons: [
+          ...(bridgeOutputs.primaryAction
+            ? [
+                {
+                  type: ButtonTypeEnum.PRIMARY,
+                  content: bridgeOutputs.primaryAction.label,
+                  url: bridgeOutputs.primaryAction.url,
+                },
+              ]
+            : []),
+          ...(bridgeOutputs.secondaryAction
+            ? [
+                {
+                  type: ButtonTypeEnum.SECONDARY,
+                  content: bridgeOutputs.secondaryAction.label,
+                  url: bridgeOutputs.secondaryAction.url,
+                },
+              ]
+            : []),
+        ],
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
💭 **thought (non-blocking):** This verbose logic is duplicated 3 times, so likely a good candidate for DRY extraction‏

Reply:
I agree. I've extracted to a shared util.